app/assets/stylesheets/shared/_surveys.scss

```diff
@@ -0,0 +1,71 @@
+.survey {
+  $base-border-radius: 3px;
```

Comment:
Rule sets should be ordered as follows: @extends, @includes without @content, properties, @includes with @content, nested rule sets